deployment/computer-vision/base64/predict.py

```diff
@@ -0,0 +1,86 @@
+import base64
+import configparser
+import multiprocessing as mp
+import numpy as np
+import os
+import pandas as pd
+import time
+import json
+
+from verta import Client
+
+
+def load_imgs(n_imgs):
```

Comment:
Loading all images in memory will take up a lot of RAM and you will run OOM. You'll need to load only the ones you are predicting on (e.g., in the parallel processing loop)

Reply:
Thanks. As we spoke yesterday, I'll work on that in another PR.